py3status/modules/yubikey.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+"""
+Show an indicator when YubiKey is waiting for a touch.
+
+Configuration parameters:
+    cache_timeout: How often to detect a pending touch request.
+        (default 1)
+    format: Display format for the module.
+        (default '{state}')
+    state_off: Message when the touch is not needed.
+        (default '')
+    state_on: Message when the YubiKey is waiting for a touch.
+        (default 'Y')
+    u2f_keys_path: Full path to u2f_keys if you want to monitor sudo access.
```

Comment:
This could be generic. `key_path`.

Reply:
I'd rather keep it like this, because this is the file name used by [pam-u2f](https://github.com/Yubico/pam-u2f) and all online tutorials use this name. If I saw `key_path` option I would never guess that I need to provide path to `u2f_keys` file

Reply:
No problem. The `pam-u2f` link seems to use `authfile`. If we parse `key_path` option, then we probably did parse the description too. Also... Using `auth_file` or `auth_key` would put it on top of the config list. Just stating things.

I wonder... I see that `authfile` points to `$HOME/.config/Yubico/u2f_keys`. Is that your path too? And if it is useful to default `$HOME/.config/Yubico/u2f_keys`?

Reply:
Yes I use the default, and thanks for the hint with `os.path.expanduser`, I thought that it will expand `~` to `/root`, but no, it expands to my own user - sweet! `$HOME` is not being expanded, only `~` by the way.

Although they use `authfile` in their README, the rest of the internet refers to this as `u2f_keys`, so I'll still keep the configuration name. Thought hopefully most people don't need to configure this anymore :)